### DISCUSSION TITLE
Just a query

### DISCUSSION THREAD

**Discussion by anandspathak** [jgromes/RadioLib#515] [General]
Sir,
I have 433MHz 100 Meters STX882 ASK Transmitter Module + SRX882 Superheterodyne Receiver Module + Antenna. Pls see at this link:

https://www.electronicscomp.com/433mhz-100-meters-stx882-ask-transmitter-module-srx882-superheterodyne-receiver-module-antenna?gclid=CjwKCAjwgr6TBhAGEiwA3aVuIZ0tWMwGjGgrkdw55HnH4InMgMwYajmJbue5Mh56YS7pFZSKKHQiMxoCq2kQAvD_BwE

Is this module supported??? 

By the way its more than two years and your library of lora SX 1278 is working great without any error...
Waiting for reply,

Thanks ,
Regards
Anand Pathak
India

**Comment by jgromes** [comment]
That looks like a simple OOK transmitter/receiver. This type of devices is unfortunately not supported.

**Reply by anandspathak** [reply]
Ok..thanks....
